src/jit/instr.h

```diff
@@ -285,15 +285,8 @@ enum InstructionSet
     InstructionSet_SSSE3   = 4,
     InstructionSet_SSE41   = 5,
     InstructionSet_SSE42   = 6,
-    InstructionSet_SSE3_4  = 7,    // SSE3, SSSE3, SSE4.1 and SSE4.2 instruction set
-    InstructionSet_AVX     = 8,
-    InstructionSet_AVX2    = 9,
-    // Linear order end
-    // TODO - Instruction sets have the linear order only in above area.
-    // We should no long compare the return value of getSIMDInstructionSet()
-    // or getFloatingPointInstructionSet() to the InstructionSet values.
-    // Should refactor SIMD code only to be aware of SIMD feature levels 
-    // (SSE2, SSE3_4, AVX, and AVX2, etc.) rather than concrete ISA.
+    InstructionSet_AVX     = 7,
+    InstructionSet_AVX2    = 8,
```

Comment:
You should delete the "// Linear order start" comment. But maybe you could add a comment about why 1-8 are different from >=32, and how InstructionSet is intended to be used.

Reply:
Will do, thanks.

Reply:
I think there should still be a comment above the "linear order" values; otherwise the reference to "gen-to-gen order" almost seems to apply to the values below it.